recipes-kernel/linux/linux-yocto-kitra710.bb

```diff
@@ -0,0 +1,20 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+LINUX_VERSION = "4.1.15"
```

Comment:
Indeed but this is the version the kitra was developed on. They are using a fork.